README.md

```diff
@@ -64,3 +64,25 @@ sudo snap connect charmhub-lp-tool:password-manager-service
 
 > **_Note:_** the `password-manager-service` interface is used to store the
 > Launchpad access token.
+
+## Configuration
+
+charmhub-lp-tool reads the configuration file from the following locations:
+
+* `~/.config/charmhub_lp_tools/config.yaml`: when installed via pip
+* `~/snap/charmhub-lp-tool/current/.config/charmhub_lp_tools/config.yaml`: when
+  installed via snap
```

Comment:
This isn't strictly true.  It actually goes for:

* `${XDG_CONFIG_HOME}/.config/charmhub_lp_tools/config.yaml`
* `${HOME}/.config/charmhub_lp_tools/config.yaml`

If running in a snap, I believe that `HOME` is set to `${HOME}/snap/....` in the context of the running snap executable?

Reply:
Fixed - https://github.com/openstack-charmers/charmhub-lp-tools/compare/df1b1b66f619464f1f765a866093afc25d69556d..2753ad6e3dbbe8f6e9114685ff72c4ef0f04c8b7